cloel.el

```diff
@@ -239,6 +252,15 @@
                                        (format "*cloel-%s-client*" app-name)
                                        host port-num)))
     (cloel-set-app-data app-name :tcp-channel channel)
+
+    ;; TODO: this is ugly
+    ;; use channel as server-process when port file exists
+    ;; so that process-live-p against :server-process works
+    (let ((app-data (cloel-get-app-data app-name)))
+      (when (and (plist-get app-data :port-from-file)
+                 (not (plist-get app-data :server-process)))
+        (cloel-set-app-data app-name :server-process channel)))
```

Comment:
I added another commit to put the channel at `:server-process` when port file exists
So that `(process-live-p server-process)` works
I'm pretty sure there are better ways to archive this so added a TODO here